edgelet/iotedge/src/check/mod.rs

```diff
@@ -849,24 +824,5 @@ mod tests {
             CheckResult::Ok => (),
             check_result => panic!("parsing {} returned {:?}", filename, check_result),
         }
-
```

Comment:
The whole test isn't needed any more. Same for the other one you deleted parts of.